server/src/database/scripts/sync-chefs-data.service.ts

```diff
@@ -56,12 +72,31 @@ export class SyncChefsDataService {
     // Maybe there's a better way to check it
     const files = objects.filter((obj) => 'url' in obj && 'data' in obj);
 
+    // Axios stuff
+    const method = REQUEST_METHODS.GET;
+    // Make sure you include the -- token=<token> into the script args
+    const token = this.getTokenFromArgs(process.argv);
+    const headers = {
+      'Content-Type': 'application/x-www-formid-urlencoded',
+      Authorization: `Bearer ${token}`,
+    };
+    const responseType = AxiosResponseTypes.BLOB;
+    const options = {
+      method,
+      headers,
+      responseType,
+    };
+
     for (const file of files) {
+      // Get file data form server
+      const url = FILE_URL + file.url;
+      const fileRes = await axios({ ...options, url });
+      const fileData = Buffer.from(fileRes.data);
+
       const newAttachmentData = {
         id: file.data.id,
         url: file.url,
-        // Data's empty for now
-        data: '',
+        data: fileData,
```

Comment:
Could you also store the name of the file!

Reply:
Added originalName to the entity and script